integration/kubernetes/k8s-nested-configmap-secret.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2021 IBM Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	export KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
+	get_pod_config_dir
+}
+
+@test "Optional and Empty Secret Volume for a pod" {
+	configmap_name="config"
+	secret_name="secret"
+	pod_name="nested-configmap-secret-pod"
```

Comment:
These are globals, so why not put them in `setup()`, or just set them at the top of the file?